stdlib/number.gr

```diff
@@ -864,6 +864,33 @@ provide let atan = angle => {
   return WasmI32.toGrain(newFloat64(WasmF64.copySign(z, sx))): Number
 }
 
+/**
+ * Computes the angle between the positive x-axis and the ray from the origin to the point (x, y).
+ *
+ * @param y: The given y coordinate
+ * @param x: The given x coordinate
+ * @returns The angle in radians between the positive x-axis and the point (x, y)
+ *
+ * @example Number.atan2(0, 1) == Number.pi
+ *
+ * @since v0.6.0
+ */
+provide let atan2 = (y, x) => {
+  if (x > 0) {
+    atan(y / x)
+  } else if (x < 0 && y >= 0) {
+    atan(y / x) + pi
+  } else if (x < 0 && y < 0) {
+    atan(y / x) - pi
+  } else if (x == 0 && y > 0) {
+    2 / pi
```

Comment:
According to wikipedia this should be `pi/2`; the one below should also be `-pi/2`

Reply:
Thanks for catching that.